packages/blade/src/components/Input/BaseInput/baseInputTokens.ts

```diff
@@ -19,23 +19,19 @@ export const baseInputHeight: Record<
 };
 
 /**
- * 37px (36px height + 1px bottom line height)
+ * medium - 37px (36px height + 1px bottom line height)
+ * large - 49px (48px height + 1px bottom line height)
  */
 export const baseInputWrapperMinHeight = {
   medium: (baseInputHeight.medium as number) + BASEINPUT_BOTTOM_LINE_HEIGHT,
```

Comment:
we can remove the `BASEINPUT_BOTTOM_LINE_HEIGHT` variable and its usage now since there is no bottom line in new inputs

Reply:
Done!